### ISSUE TITLE
Changing text in finder competency viewer to not be "Competencies"

### ISSUE THREAD

**Issue by stuartasutton** [CredentialEngine/Credential-Finder#2]
@siuc-nate & @jkitchensSIUC:

In the finder view of competency frameworks, below the framework description and other header information, there is a light grey header bar with the text "Competencies". We need to think of an alternative to "Competencies" since the load of identifying nodes in the framework are handled through the competencyCategory since the framework may be filled with identified nodes such as "Skill", "Ability", "Task", "Benchmark" etc., etc., etc.  Just to prime the pump, perhaps something like "Framework Content"?

Stuart

**Comment by siuc-nate** [comment]
Possibly, though in our CASS meeting the other day with @Lomilar's team, we discussed using that very header to differentiate between the competency editor and the concept scheme editor (and putting the two editors on different but functionally identical pages to avoid a lot of complexity on the back-end).

That said, it's possible that we may want to enable publishing either competency frameworks or concept schemes from the same page, so that plan might not even be viable.  

Would this also imply a need to remove the word "competency" anywhere else it appears (labels, URLs, etc)? That could get tricky.

**Comment by stuartasutton** [comment]
Nate. I was talking about the competency **viewer** in the Credential Finder and not the editors.

**Comment by stuartasutton** [comment]
No! I am just talking about general public facing stuff like display headers etc.

**Comment by siuc-nate** [comment]
> Nate. I was talking about the competency **viewer** in the Credential Finder and not the editors.

D'oh. I missed that.  Though that brings up a different thought: would we need to call the "competency framework" search something else?

**Comment by stuartasutton** [comment]
No, when we are looking across all the potential competencyCategories we'll see, I still think Competencies is the best umbrella.

**Comment by siuc-nate** [comment]
What should the text of that bar say instead?
